### DISCUSSION TITLE
[SF]Search icon with blue background

### DISCUSSION THREAD

**Discussion by tandan0813** [UI5/webcomponents#5474] [Q&A]
Hi, collegue,
I'm a member of WEF ShangHai team, and our UX gave a design spec that there's a blue background for search icon when there is content in the search field(screenshot attached). Just curious if we have that design in FN for search icon? And if not, do we have a plan?
Thanks!
<img width="436" alt="image" src="https://user-images.githubusercontent.com/10113427/177675099-34891dc4-349e-4ffc-8013-4ea0e6bf6b38.png">

**Comment by MarcusNotheis** [comment]
I've just transferred this discussion from the Web Components for React repo - looks like a duplicate of #5472?

**Reply by tandan0813** [reply]
Yes, it's duplicate. Thanks!
